pages/clients/cli.js

```diff
@@ -58,6 +58,7 @@ export default () => (
       <code>
 {`{
   "platform": "node",
+  "project": "my-project"
```

Comment:
I think we need a `,` at the end of this line.

Reply:
My bad :blush: